cli/types/cypress.d.ts

```diff
@@ -24,9 +24,15 @@ declare namespace Cypress {
   interface CommandFn<T extends keyof ChainableMethods> {
     (this: Mocha.Context, ...args: Parameters<ChainableMethods[T]>): ReturnType<ChainableMethods[T]> | void
   }
+  interface CommandFnArgs<T extends keyof ChainableMethods> {
+    [name: string]: (this: Mocha.Context, ...args: any) => (ReturnType<ChainableMethods[T]> | void)
```

Comment:
At first, I wrote it this way, because `...args: Parameters<ChainableMethods[T]>` doesn't work. TypeScript tries to interpret `args` as the union type of all possible `cy` method arg types (i.e. [type of cy.get parameters] | [type of cy.visit parameters] | etc... ). 

This happened because `<T extends keyof ChainableMethods>` part defined after the `interface` means that this `CommandFnArgs` interface should use the parameters of one of `ChainableMethods` (i.e. one of the name of `cy` methods) for all of its functions. It's not the right behavior. 

So, I thought about using generics inside `[key]` brackets. I realized that the purpose of `addAll` is to add a **new** command. We're not reusing any `cy` command. They should be `any`, because we don't know what they are. They're defined by users.

Reply:
Gottchya. That makes sense! Thank you for the explanation!